apps/greenhouse-management/src/lib/store.js

```diff
@@ -3,6 +3,8 @@ import { devtools } from "zustand/middleware"
 import { managementPluginConfig } from "../../package.json"
 import { useActions as messageActions } from "messages-provider"
 
+const isProduction = process.env.NODE_ENV === "production"
```

Comment:
We can't use process.env.NODE_ENV as an environment. As soon as the application is build the environment will be allways production. We wouldn't have any difference then between QA and PROD since in both cases we have build the application and it is production.

Reply:
we are able already to pass the env per Props:
https://github.com/sapcc/juno/blob/main/apps/greenhouse-management/package.json#L103

Reply:
and environment is also passed from greenhouse to the greenhouse-mng:
https://github.com/sapcc/juno/blob/main/apps/greenhouse/src/lib/plugin.js#L91

Reply:
:100: 

changed